.github/ISSUE_TEMPLATE/release_new_version.md

```diff
@@ -0,0 +1,70 @@
+- [ ] [Tag a beta release](https://github.com/jamulussoftware/jamulus/blob/master/RELEASE-PROCESS.md#steps-for-a-specific-release)
+- [ ] Announce the beta release on Github Discussions. Pin the thread.
+- [ ] Get feedback on the stability of the beta release
+- [ ] Finish Website translations
+  - [ ] Wait for all PRs to be merged
+  - [ ] Decide what to do about missing translations. Depending on the missing changes, consider keeping the old state (if it is still valid) or temporary backing out the relevant file(s) if they would become wrong.
+  - [ ] Merge `translate3_y_z` into `changes` via PR
+- [ ] Finish App translations
+  - [ ] Review PRs <details><summary>Checklist for the PRs</summary> <!-- FIXME: This should be moved to the admin wiki and linked here -->
+     ```
+     - [ ] Translator listed in the `src/util.cpp` **optionally add link to PR or code**
+     - [ ] Punctuation and spacing consistent
+     - [ ] Signal words consistent ("ASIO", "Buffer")
+     - [ ] App translations: No untranslated strings (`grep unfinished -5 src/res/translation/translation_$TRANSLATION*.ts`)
+     - [ ] App translations: Only a single `.ts` file checked in (`.qm` in addition is also OK)
+     - [ ] Installer translations: Passes `tools/check-wininstaller-translations.sh`
+     ```
+     </details>
+  - [ ] Wait for all PRs to be merged. If translations are missing after the targeted translation completion date, this is ok. They can be updated with the next release again.
+  - [ ] Check for conflicting accelerator keys (see `tools/checkkeys.pl`)
+  - [ ] Generate `.qm` files via `lrelease Jamulus.pro`
+- [ ] [Tag a release candidate](https://github.com/jamulussoftware/jamulus/blob/master/RELEASE-PROCESS.md#steps-for-a-specific-release)
+- [ ] Announce the release candidate on Github Discussions. Pin the thread. Unpin and lock the beta thread.
+- [ ] Draft an announcement, include all contributors via `tools/get_release_contributors.py`
+- [ ] [Update the version number in `Jamulus.pro` and add the release date to the Changelog header and commit](https://github.com/jamulussoftware/jamulus/blob/master/RELEASE-PROCESS.md#steps-for-a-specific-release)
+- [ ] Tag this commit as `r3_y_z`
+- [ ] [Force tag that tag as `latest` and push.](https://github.com/jamulussoftware/jamulus/blob/master/RELEASE-PROCESS.md#if-this-is-a-proper-release-move-the-latest-tag)
+- [ ] Wait for the build to complete
+- [ ] [Prepare `Jamulus.pro` (`dev` suffix) and ChangeLog (add a header) for the next release](https://github.com/jamulussoftware/jamulus/blob/master/RELEASE-PROCESS.md#make-the-master-branch-ready-for-post-release-development)
+- [ ] Do a smoke test for Windows/Mac/Linux -- Do the binaries start/connect properly?
```

Comment:
Smoke test should come before tagging as latest - i.e. off the r3_x_y build once it's been created.  Those are the "final" binaries and the smoke test should come as soon as possible - in case it's not actually final! :)